web/controllers/screeningController/index.ts

```diff
@@ -245,7 +247,15 @@ export async function updateStudentByMailHandler(req: Request, res: Response, ne
     if (studentInfo.isValid()) {
         //save old state of student info
         const prevState = await StudentInfoDTO.buildFrom(student);
-
+        
+        // Check if email already exists
+        const email_dupe_res = await entityManager.findOne(Student, {email: studentInfo.email.toLocaleLowerCase()});
```

Comment:
This will find the student we are editing unless we don't change the email address und will thus throw an error in all cases in which we edit something but the mail address 😉